sycl/test/launch_bounds/lb_sm_70.cpp

```diff
@@ -0,0 +1,67 @@
+//  REQUIRES: cuda
+
+// RUN: not %clangxx -ferror-limit=100 -fsycl -fsycl-targets=nvptx64-nvidia-cuda -Xsycl-target-backend --cuda-gpu-arch=sm_70 -fsyntax-only %s -o - 2>&1 | FileCheck %s
+
+// NOTE: we can not use the `-verify` run alongside
+// `expected-error`/`expected-warnings` as the diagnostics come from the device
+// compilation, which happen in temporary files, while `expected-...` are
+// placed in the main file, causing clang to complain at the file mismatch
```

Comment:
Can we use `fsycl-is-device` (or whatever the spelling is)? Also, I'd expect it's the host compilation that's causing the problems, not the device as the latter is compiled unmodified.

Reply:
Also, those tests should probably go to `clang/test/`

Reply:
You're rights `-fsycl-device-only` does the trick. Also moved the tests to `clang/tests/SemaSYCL`, in a625b6af301d394fcb6639cc361060958f5bd045. Thanks.